src/test/java/org/folio/util/TagsTestUtil.java

```diff
@@ -1,7 +1,20 @@
-package org.folio.rest.impl;
+package org.folio.util;
 
-import static org.apache.commons.lang3.StringUtils.join;
+import io.vertx.core.Vertx;
+import io.vertx.core.json.JsonArray;
+import io.vertx.core.json.JsonObject;
+import io.vertx.ext.sql.ResultSet;
+import org.folio.rest.persist.PostgresClient;
+import org.folio.tag.RecordType;
+import org.folio.tag.Tag;
 
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.UUID;
+import java.util.concurrent.CompletableFuture;
+
+import static org.apache.commons.lang3.StringUtils.join;
```

Comment:
@stanislav-mureiko -- this file as well as others only have changes due to ordering of imports -- I don't believe this ordering should be changed. If using import ordering configuration files for Intellij (provided by @dmtkachenko) -- order will and should not change

Reply:
Thanks, @carolegodfrey -- I believe I was using configuration files for Intellij provided by @dmtkachenko. But I need to double check it.

Reply:
I've fixed it in another PR https://github.com/folio-org/mod-kb-ebsco-java/pull/141

Reply:
Ok thanks @stanislav-mureiko